doc/help/settings.asciidoc

```diff
@@ -1183,6 +1185,14 @@ Valid values:
 
 Default: +pass:[middle]+
 
+[[tabs-pinned-width]]
+=== pinned-width
+The width for pinned tabs with a horizontal tabbar, in px.
+
+See `:tab-pin` for more details.
+
+Default: +43+
```

Comment:
Is setting this in `px` going to be a problem for HiDPI devices? I don't know how qt handles HiDPI...